### ISSUE TITLE
[j5.1][bug Dark theme] Styles error for a dark theme

### ISSUE THREAD

**Issue by korenevskiy** [joomla/joomla-cms#43333]
![изображение](https://github.com/joomla/joomla-cms/assets/6898474/5f4942b4-3581-464f-aecd-905c796f94f1)
According to the rules of the Bootstrap button design styles, the buttons can have a full color fill or OUTLINE.

So the Outline style doesn't work.
We can see how the main color for a regular button differs from the color of the Outline button.

**Comment by exlemor** [comment]
Hi @korenevskiy, isn't this an issue with the extension developer and how the programmed that button vs a Joomla one? 

(just asking/trying to understand for my own knowledge)

**Comment by korenevskiy** [comment]
```HTML
<button  class=" btn btn-outline-primary btn-sm" type="button" id="camera_start_110">Camera Start</button>
```
the truth is somewhere in between. Of course, this does not affect the Admin interface. But does the component developer have the right to use the style of the btn-outline-primary and btn-outline buttons?
Since the template is based on Bootstrap. This means that the template should allow you to use a grid from Bootstrap, control styles, calendars and others.

**Comment by brianteeman** [comment]
> Since the template is based on Bootstrap. 

Based on bootstrap != is bootstrap

**Comment by korenevskiy** [comment]
You're right, but let's evaluate. Joomla should be treated as just a CMS without frameworks or a CMS with frameworks and libraries. So that you can rely on them

**Comment by korenevskiy** [comment]
And then it turns out that the programmer needs to connect the style to the admin panel.

**Comment by coolcat-creations** [comment]
Since Scanner QR Tickets is not a core module it seems the wrong classes have been used there and the issue might be closed? @Hackwar

**Comment by korenevskiy** [comment]
> Since Scanner QR Tickets is not a core module it seems the wrong classes have been used there and the issue might be closed? [@Hackwar](https://github.com/Hackwar)

Let me remind you again. I'm thinking the problem here is not that Bootstrap dark theme is not supported. And the fact is that Joomla as a CMS should support Bootstrap so that developers can rely on it, or Joomla should not support bootstrap, and this is just a feature of the template. This is a matter of Joomla philosophy, not a bug in the code.

**Comment by brianteeman** [comment]
Core bootstrap - screenshot from their documentation

![Image](https://github.com/user-attachments/assets/327d020b-51cb-45e3-8858-919906c940ab)

Same code in a module in j5 admin

![Image](https://github.com/user-attachments/assets/ea648c4b-d131-4694-b35e-3804596e8748)


While I accept that the colours may vary slightly the css in joomla is clearly not providing the expected styles. This is not the only place that core bootstrap markup does not work due to errors in the joomla css as we have the exact same problem in https://github.com/joomla/joomla-cms/issues/43708

**Comment by korenevskiy** [comment]
Don't get me wrong.
It's not that I'm styling a dark theme and can't use a secondary button instead of a primary button. I can make another button.
But the fact is that when the user uses a light theme, he should see the main button, I make it "Primary", but the user switches the theme to a dark one and the button becomes INVISIBLE. It turns out that I need to track  the moment of switching themes so that the main button can change button classes.
It turns out that to be sure of the color, only half of all the OUTLINE buttons can be used: Success, Danger, Warning, Info.
I will use the button for these purposes. The Primary and Secondary buttons are not applicable by definition. Since they are either not visible, or they become brighter than the main button.
Let's close this problem, here we need to make amendments to the built-in Bootstrap framework to correct the BTN-OUTLINE, or not distract attention.
**Let's vote.** 

- Like - requires correction, 
- Dislike - Does not require correction.

**Comment by Quy** [comment]
Related #43131

**Comment by jjnxpct** [comment]
When we use the light theme the backend 'login with passkey' button is coded to have btn-secondary. But the base color is blue (like the primary) and the hover is a white background and white icon.  When this button css is changed to 'btn-primary' it work OK like a primary button. My point: The btn-secondary (hover) is not working OK here. I assume this is the same issue as presented here.

I am not sure the creators of the backend template meant to actually have a 'btn-secondary' here but I assume they did. So thing the button needs to be grey (not blue) with a slightly darker gray hover.

![Image](https://github.com/user-attachments/assets/da559e4a-46f1-4a09-8922-2d4c3a92e650)

I think having this button as 'secondary' is a good way to style this button, as long as the classes for the secondary button are working in the backend.

When I am wrong about this being the same issue, please let me know so I can create a separate issue for this. Thakks!
